src/docs/concepts-users-and-authentication/user-permissions/index.md

```diff
@@ -98,12 +110,160 @@ Now, you can check or uncheck the permissions checkbox for this user.
 
 ![User permissions checkbox](./user_publish_listings_checkbox.png)
 
+See which endpoints are affected by this setting
+[here](#restrict-publishing-rights).
+
+Read more about this feature in the
+[Help Center](https://www.sharetribe.com/help/en/articles/9503118-restrict-listing-publishing-rights).
+
+## Restrict transaction rights
+
+You might also want to restrict the ability to initiate transactions to
+specific users only. For example, if you have a marketplace where users
+can view listings but need to verify their identity before making
+purchases, you can enable this feature to manually control who can
+initiate transactions. Similarly, if your platform uses a subscription
+model, you may want to allow only paying users to start transactions. In
+some cases, you may run a marketplace that functions primarily as a
+showroom, where users can browse but cannot buy until the operator
+grants them permission.
+
+You can toggle the setting in the Access control tab.
+
+![Restrict transaction rights checkbox](./users_restrict_transaction_rights.png)
+
+Once this checkbox is selected, similiarly to how publishing rights are
+granted, you can see the permission status of each user in the Console's
+Manage > Users view.
+
+See which endpoints are affected by this setting
+[here](#restrict-transaction-rights-1).
+
+Read more about this feature in the
+[Help Center](https://www.sharetribe.com/help/en/articles/9790336).
+
+## API level restrictions
+
+Toggling permissions under the "Access Control" tab in the Console
+directly impacts API behavior. When permissions are modified, endpoint
+access is adjusted accordingly, ensuring that users cannot bypass the
+restrictions imposed by these settings. This guarantees access control
+policies are strictly enforced. These restrictions only apply in the
+Marketplace API.
+
+### Making marketplace private
+
+Toggling this setting in Console requires users to log in to access
+listings.
+
+Making the marketplace private from Console will restrict the following
+endpoints:
+
+- `GET /users/show`
+- `GET /listings/query`
+- `GET /listings/show`
+- `GET /timeslots/query`
+- `GET /reviews/query`
+- `GET /reviews/show`
+- `GET /sitemap_data/query_listings`
+
+When this setting is toggeld, these endpoints will return a 403
+Forbidden response, indicating that access is denied due to the
+marketplace's private status.
+
+### Approve users who want to join
+
+Toggling this setting in Console will require users to be approved by an
+operator to join the marketplace.
+
+This will restrict the following endpoints:
```

Comment:
User approval has slightly different behavior based on whether the marketplace is public or private – this part could refer to the private marketplace endpoints as well somehow, since they're the same for unauth and unapproved users.